metaconfig-pprint/shared/src/main/scala-2/metaconfig/pprint/TPrintImpl.scala

```diff
@@ -0,0 +1,274 @@
+package metaconfig.pprint
+import language.experimental.macros
+import reflect.macros.blackbox.Context
+
+trait TPrintLowPri {
```

Comment:
I only inlined TPrint part, but maybe we should inline it all? This way we could drop the dependency on pprint in mdoc too and this has been causing issues.